packages/AsyncLoader/README.md

```diff
@@ -35,9 +35,12 @@ Component that fetches contents asynchronously and displays it when ready:
 `<async-loader></async-loader>`
 
 #### Attributes
-- `data-endpoint-url` (required): URL that should be fetched
+- `data-endpoint-url` (required): URL that should be fetched.
```

Comment:
Ich würd's grad so machen: `data-endpoint-url` (required if `data-event-endpoint-property-name` is not set): URL that should be fetched. Dito unten. Weil «(required)» ohne weitere Bedingung ja nicht korrekt ist.